womtool/src/test/resources/validate/wdl_draft3/invalid/missing_subworkflow_input/missing_subworkflow_input.inputs.json

```diff
@@ -0,0 +1,3 @@
+{
+  "missing_subworkflow_input.subwf.i": 5
+}
```

Comment:
Missed a newline.

FYI I've stopped requesting newlines on `*.scala` only because I'm waiting for ScalaFmt to fix all. 🙏  

However we don't have a similar linter for json, well except [IntelliJ](https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file#42822895).

Reply:
I wish I'd found out about that option three years ago! Now enabled!